project/ProjectPlugin.scala

```diff
@@ -28,7 +28,7 @@ object ProjectPlugin extends AutoPlugin {
       addSbtPlugin("org.tpolecat"     % "tut-plugin"  % "0.5.2"),
       libraryDependencies ++= Seq(
         %%("moultingyaml"),
-        "com.47deg"             %% "org-policies-core" % "0.5.8",
+        "com.47deg"             %% "org-policies-core" % "0.5.15",
```

Comment:
The last published version is `0.5.14`

Reply:
Bumped tut-plugin again for scala-xml:1.06: https://github.com/47deg/sbt-org-policies/pull/487, which you just approved as I was typing this comment.